script.js

```diff
@@ -0,0 +1,78 @@
+const searchButton = document.getElementById("globalSearchBtn");
```

Comment:
if this file isn't in use -remove it. Don't worry, your code is saved inside git history (look for in inside github)